### ISSUE TITLE
💡 [FEAT] better documentation ->  fonts and config files

### ISSUE THREAD

**Issue by JindrazPrahy** [dooit-org/dooit#119]
I had trouble understanding how to set up dooit, namely with two things (and I still am not sure if I understand it)

1. Getting the fonts to work

Emojis make up the graphics in dooit, so I think they are integral to the program. Yet fonts aren't in the dependencies (for understandable reasons). Still, I think it would be good if the project indicated (i. e. in the README) which font is preferred and perhaps how to make the emojis work in a terminal setting, where it's monospace. (While looking through the source code I found a comment saying nerd fonts are preffered? Still, I don't think reading the source code should be required for users.)

2. Understanding how to change the keybindings

My current understanding is that keybindings are stored in a file called config.py? It's highly unusal for a config file to be a .py file. Imo this should be stated in the wiki:
https://github.com/kraanzu/dooit/wiki

I also think it might be a good idea to fill the config.py with the default keybindings so that the user can just easily change them and doesn't have to worry about the syntax.

To sum up, this feature request is to somehow indicate how to do those things somewhere in the project so that it's easy to find.
It might be even considered a bug report.

**Comment by kraanzu** [comment]
Hi @JindrazPrahy ,

Thanks for opening the issue!

>  Getting the fonts to work

In the previous version, I directly used nerd fonts because I didn't think many people will use it haha but this issue was raised time and again so I replaced all the default icons with unicode icons from [this site](unicode-table.com)  which I think should be present by default in any system?

> It's highly unusal for a config file to be a .py file.

Yes I guess that might be true

> I also think it might be a good idea to fill the config.py with the default keybindings

I think this can be a great idea. Maybe you can do something like `dooit example` which will create a default config with all the default settings in it!?

**Comment by kraanzu** [comment]
I understand! Thanks a lot for the detailed explanation!
I'll do the implementations!

I'm currently working on optimizing and refactoring the UI so I'll start working on this a few days later!

**Comment by Justinzobel** [comment]
I am also finding it hard to figure out how to enable the dashboard for Overdue/Current/Done tasks.

**Comment by kraanzu** [comment]
Hey @Justinzobel, can you explain? Do you mean the details in the bottom bar?

**Comment by Justinzobel** [comment]
> Hey @Justinzobel, can you explain? Do you mean the details in the bottom bar?

This one https://user-images.githubusercontent.com/97718086/221467485-fae198f7-51b1-4a71-91d9-88b51897aeeb.png is what I'm trying to achieve but even with 4 workspace setup, some tasks set as complete, some with due dates it doesn't show up. I think I have to add it to a config but I'm confused by the instructions.

**Comment by kraanzu** [comment]
Ah yes, currently you have to do a lot manually to get the count of tasks. I'll try to implement something which will make it easier to fetch and filter them

**Comment by kraanzu** [comment]
Closing this as now dooit uses regular unicode and has a separate repo for extra eye candy!
